src/locale.ts

```diff
@@ -136,7 +136,7 @@ export let array: Required<ArrayLocale> = {
 export let tuple: Required<TupleLocale> = {
   notType: (params) => {
     const { path, value, spec } = params;
-    const typeLen = spec.types.length;
+    const typeLen = spec.innerType.length;
```

Comment:
i'd rather keep the `types` name, i wanted to migrate array to use it it as well but opted to avoid the breaking change

Reply:
OK. I can change that back to `types`. I suppose that is actually a breaking change too, so it should remain `types`. Its a bummer I didn't get to this before the official release of V1 🤷 

What should I do with Array?
1 - Return it to its current shape (no spec)
2 - Add `spec.innerType = {...the one type}`
3 - Add `spec.types = [{ ...the one type }]` // note the array
4 - Add `spec.types = { ...the one type }` // note no array
5 - Add `spec.type = { ...the one type }` // note spec is singular

I am leaning towards #3. It seems most consistent with Tuple. And if there is ever another use case for some "inner type", then the property `types` with an array is flexible.

Reply:
I think 3 makes sense to me 👍